src/index.html

```diff
@@ -10,7 +11,36 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <!-- #endregion -->
+  <body class="body">
+    <form
+      action="/"
+      method="post"
+      class="form"
+      data-qa="big"
+    >
+      <label for="input-big" class="icon icon--big"></label>
+      <input
+        type="text"
+        class="input form__input input--size--big"
+        id="input-big"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
+    <form
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML
```suggestion
    </form>
    
    <form
```